waspc/data/Generator/templates/server/src/core/auth.js

```diff
@@ -47,14 +45,7 @@ const auth = handleRejection(async (req, res, next) => {
 })
 
 export const createNewUser = async (userFields) => {
-  const hashedPassword = await hashPassword(userFields.password)
-
-  const newUser = await prisma.{= userEntityLower =}.create({
-    data: {
-      ...userFields,
-      password: hashedPassword
-    },
-  })
+  const newUser = await prisma.{= userEntityLower =}.create({ data: userFields })
```

Comment:
Hm, so does this actually make this `createNewUser` method redundant? It does nothing but call prisma.User, right? And it complicates things since we have to know about it. So maybe we could remove it?

Reply:
True, opened issue for it as we agreed.`